pages/content/amp-dev/about/websites-2021.html

```diff
@@ -1,10 +1,11 @@
 ---
 $title: AMP Websites
 $view: /views/2021/base.html
-description: 'AMP is a simple and robust format to ensure your website is fast, user-first, and makes money. AMP provides long-term success for your web strategy with distribution across ...'
+description: 'AMP is a simple and robust format to ensure your website is fast, user-first, and makes money. AMP
```

Comment:
And the new line here is wrong. Probably your auto-formatting does weird things here.